packages/core/database/src/index.ts

```diff
@@ -119,6 +124,15 @@ class Database {
     this.entityManager = createEntityManager(this);
   }
 
+  async repair(options: RepairOptions) {
```

Comment:
An alternative would be do pass in these options when calling `sync` and sync could do the normal db sync, then follow up with this

Reply:
But this method _should_ (untested) work for any polymorphic relations table, so it could be useful for also cleaning up deleted content types.

I didn't attempt to clean everything because it would have been morecomplex to write the code to define what needs to be deleted, and I was also worried about potentially causing data loss or a performance impact.